prometheus/prometheus.yml

```diff
@@ -1,17 +1,29 @@
 global:
-  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
-  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  scrape_interval:     12s # Set the scrape interval to every 12 seconds. Default is every 1 minute.
```

Comment:
12s because of slot times ?

Reply:
Yes, otherwise every now and again 1 scrape has counter values jump by two (e.g. slot number)